rootfs/api/models/release.py

```diff
@@ -218,6 +219,9 @@ def rollback(self, user, version=None):
                 raise DeisException('Cannot roll back to initial release.')
 
             prev = self.app.release_set.get(version=version)
+            if prev.failed:
+                raise DeisException('Cannot roll back to failed release.')
```

Comment:
I don't agree with this. Users should be able to roll back to a failed release, as a release can fail due to "an act of god" upon the app, rather than an app failing to deploy because the user unintentionally fat-fingered the wrong image name.